src/main/java/com/epam/lab/paymentsystem/service/AccountService.java

```diff
@@ -14,7 +14,7 @@
    *
    * @return list of accounts
    */
-  List<Account> getAllAccountsOfCurrentUser();
+  List<Account> getAllAccountsOfCurrentUser(String login);
```

Comment:
If you get it by login then it is not `AccountsOfCurrentUser`. Please, rename it to `getAllAcountsOfUser`